README.md

```diff
@@ -1,9 +1,34 @@
-# Your Project's Title...
-Your project's description...
+# Target VEC for Helix
+The project is a proof of concept for integrating Target VEC with Helix.
 
 ## Environments
-- Preview: https://main--{repo}--{owner}.hlx.page/
-- Live: https://main--{repo}--{owner}.hlx.live/
+- Preview: https://main--helix-target-integration--adobe-rnd.hlx.page/
+- Live: https://main--helix-target-integration--adobe-rnd.hlx.live/
+
+## Adding Target integration to a Helix project
+1. Copy the [`scripts/target.js`](scripts/target.js) to your project.
+2. Include the script in the [`head.html`](index.html) as early as possible:
+```html
+<script src="./scripts/target.js" type="module"></script>
+```
+2. Add a call to `loadTargetOffers` to the 'loadPage' function as follows:
+```js
+async function loadPage() {
+  loadTargetOffers('{Target Client Code}');
```

Comment:
You have tight coupling here between the 2 scripts, so we actually need to make sure `target.js` is loaded _before_ `scripts.js`. And we could also instrument directly at the top of the `scripts.js` without touching the `loadPage` method

Reply:
If it is imported the module only executes if all imports are resolved. It should be fine like that.